var/spack/repos/builtin/packages/py-pkginfo/package.py

```diff
@@ -12,9 +12,9 @@ class PyPkginfo(PythonPackage):
     homepage = "https://code.launchpad.net/~tseaver/pkginfo/trunk"
     pypi = "pkginfo/pkginfo-1.5.0.1.tar.gz"
 
+    version("1.9.6", sha256="8fd5896e8718a4372f0ea9cc9d96f6417c9b986e23a4d116dda26b62cc29d046")
     version("1.8.3", sha256="a84da4318dd86f870a9447a8c98340aa06216bfc6f2b7bdc4b8766984ae1867c")
     version("1.7.1", sha256="e7432f81d08adec7297633191bbf0bd47faf13cd8724c3a13250e51d542635bd")
     version("1.5.0.1", sha256="7424f2c8511c186cd5424bbf31045b77435b37a8d604990b79d4e70d741148bb")
 
     depends_on("py-setuptools", type=("build", "run"))
```

Comment:
Is setuptools used at run-time?

Reply:
In pkginfo/tests I see `import pkg_resources`, That's why I left it.

Reply:
Wasn't sure if that was only for unit tests or not

Reply:
I was not sure either. I just tested and for me all version install with `--test=root` also if `py-setuptools` is only a build dependency. Should I remove it then?

Reply:
Yeah let's keep it build-only

Reply:
It's removed now :-)